cli/defenseclaw/tui/panels/setup.py

```
@@ -233,6 +252,8 @@ class SetupWizard(IntEnum):
     "Runs: defenseclaw setup splunk dashboards apply|destroy --yes. Requires the Splunk O11y realm + API token.",
     "Runs: defenseclaw setup trusted-paths list|add|remove. Need a directory for add/remove.",
     "Runs: defenseclaw guardrail status|enable|disable|fail-mode|hilt|block-message with optional --connector.",
+    "Runs: defenseclaw setup redaction. Quick actions are non-interactive; the guided workflow exposes every "
+    "advanced bucket, profile, destination, and ordered-route setting.",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)